code/ai/ai_profiles.cpp

```diff
@@ -582,7 +582,17 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				set_flag(profile, "$ships playing dead don't manage ETS:", AI::Profile_Flags::Ships_playing_dead_dont_manage_ets);
 
-				set_flag(profile, "$better combat collision avoidance for fightercraft:", AI::Profile_Flags::Better_collision_avoidance);
+				set_flag(profile, "$better combat collision avoidance for fightercraft:", AI::Profile_Flags::Better_combat_collision_avoidance);
+
+				if (optional_string("$combat collision avoidance aggression for fightercraft:")) {
+					stuff_float(&profile->better_collision_avoid_aggression_combat);
+				}
+
+				if (optional_string("guard collision avoidance aggression for fightercraft:")) {
```

Comment:
I would put this below the avoidance flag itself and use `+` to make it more clear and the aggression values are subordinate to the flag to use them.

Reply:
Thanks, just updated!